templates/copy-oidc-client-token.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Capabilities.APIVersions.Has "kyverno.io/v1" }}
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: "copy-oidc-client-token-for-kiali"
+  annotations:
+    # ensure that we copy the secret before kiali rollout
+    argocd.argoproj.io/sync-wave: "-5"
+    # avoid troubles with cluster-policy immutable fields
+    argocd.argoproj.io/sync-options: Replace=true
+spec:
+  background: true
+  generateExistingOnPolicyUpdate: true
+  rules:
+  - name: "copy-oidc-client-token-for-harbor"
```

Comment:
Also here: Maybe kiali instead of harbor?

Reply:
right, fixed with yaml anchors